test/component/features/steps/event_processor_steps.py

```diff
@@ -0,0 +1,114 @@
+from os import getenv
+
+from behave import given, then, when
+from features.utilities.utilities import load_json_file, read_log_file
+from requests import post
+from json import dumps
+
+
+@given("a change event that is for a service is the different from a DoS service")
+def a_change_event_that_is_for_a_service_is_the_different_from_a_dos_service(context):
+    context.change_event = load_json_file("event_processor_change_event_different_from_dos.json")
+
+
+@given("a change event that is for a service is the same as a DoS service")
+def a_change_event_that_is_for_a_service_is_the_same_as_a_dos_service(context):
+    context.change_event = load_json_file("event_processor_change_event_same_as_dos.json")
+
+
+@given("a change event for a service that doesn't exist in DoS")
+def the_change_event_for_a_service_that_does_not_exist_in_dos(context):
+    context.change_event = load_json_file("event_processor_change_event_different_from_dos.json")
+    context.change_event["ODSCode"] = "FXXX1"  # Not in DoS
+
+
+@when("the change event is sent to the event processor")
+def the_change_event_is_sent_to_the_event_processor(context):
+    """Sends change event to Event Processor lambda"""
+    lambda_url = getenv("EVENT_PROCESSOR_FUNCTION_URL")
+    context.response = post(url=lambda_url, headers={"Content-Type": "application/json"}, json=context.change_event)
+    assert context.response.status_code == 200, "Lambda emulator didn't accept request"
+
+
+@then("the change event that is for a service is the different from a DoS service is logged")
```

Comment:
`is different`, rather than `is the different`

Reply:
Updated, fixed the grammar error